app.go

```diff
@@ -387,6 +403,13 @@ func (a *Application) InitChain(req abci.RequestInitChain) abci.ResponseInitChai
 }
 
 func (a *Application) BeginBlock(req abci.RequestBeginBlock) abci.ResponseBeginBlock {
+	var err error
+	defer func(begin time.Time) {
+		lvs := []string{"method", "BeginBlock", "error", fmt.Sprint(err != nil)}
+		beginBlockLatency.With(lvs...).Observe(time.Since(begin).Seconds())
+		log.Info(fmt.Sprintf("BeginBlock took %f seconds-----\n", time.Since(begin).Seconds()))
```

Comment:
I don't want this logged, just need the metrics. Can drop the `error` label since errors in `BeginBlock` & `EndBlock` are terminal.